internal/events/eventstream.go

```diff
@@ -50,43 +50,47 @@ const (
 	ErrorHandlingSkip = "skip"
 	// MaxBatchSize is the maximum that a user can specific for their batch size
 	MaxBatchSize = 1000
-	// DefaultExponentialBackoffInitial  is the initial delay for backoff retry
+
 	DefaultExponentialBackoffInitial = time.Duration(1) * time.Second
-	// DefaultExponentialBackoffFactor is the factor we use between retries
-	DefaultExponentialBackoffFactor = float64(2.0)
-	// DefaultTimestampCacheSize is the number of entries we will hold in a LRU cache for block timestamps
-	DefaultTimestampCacheSize = 1000
+	DefaultExponentialBackoffFactor  = float64(2.0)
+	DefaultTimestampCacheSize        = 1000
+	DefaultBlockedRetryDelaySec      = 30
+	DefaultBatchTimeoutMS            = 5000
+	DefaultErrorHandling             = ErrorHandlingSkip
 )
 
+var falseValue = false
+var trueValue = true
+
 // StreamInfo configures the stream to perform an action for each event
 type StreamInfo struct {
 	eventsapi.TimeSorted
 	ID                   string               `json:"id"`
 	Name                 string               `json:"name,omitempty"`
-	Path                 string               `json:"path"`
-	Suspended            bool                 `json:"suspended"`
-	Type                 string               `json:"type,omitempty"`
+	Path                 string               `json:"path,omitempty"`
+	Suspended            *bool                `json:"suspended,omitempty"`
+	Type                 string               `json:"type"`
 	BatchSize            uint64               `json:"batchSize,omitempty"`
 	BatchTimeoutMS       uint64               `json:"batchTimeoutMS,omitempty"`
 	ErrorHandling        string               `json:"errorHandling,omitempty"`
 	RetryTimeoutSec      uint64               `json:"retryTimeoutSec,omitempty"`
-	BlockedRetryDelaySec uint64               `json:"blockedReryDelaySec,omitempty"`
```

Comment:
😄  - reminds me this one is in ethconnect, and needs a migration compatible fix there